cert-manager/cert-manager/base/api-service.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1beta1.webhook.certmanager.k8s.io
+  labels:
+    app: webhook
+    app.kubernetes.io/name: webhook
```

Comment:
you can leave out these labels - see earlier comment

Reply:
Sure